src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.Retention;
```

Comment:
Consider adding a brief description to the @Component annotation to provide more context for its purpose and usage.